app/renderer/components/common/textbox.js

```diff
@@ -77,6 +80,65 @@ class DefaultTextArea extends ImmutableComponent {
   }
 }
 
+class WordCountTextArea extends React.Component {
+  constructor () {
+    super()
+    this.handleCopyToClipboard = this.handleCopyToClipboard.bind(this)
+    this.handleOnChange = this.handleOnChange.bind(this)
+    this.state = { wordCount: 0 }
+  }
+
+  handleOnChange (e) {
+    let wordCount = 0
+
+    if (e.target.value.length > 0) {
+      wordCount = e.target.value.trim().replace(/\s+/gi, ' ').split(' ').length
+    }
+
+    this.setState({wordCount})
+
+    if (this.props.onChangeText) {
+      this.props.onChangeText()
+    }
+  }
+
+  handleCopyToClipboard () {
+    if (!this.passphraseDisplay) {
```

Comment:
we might want to call this something else, like `textAreaBox` (since it may not always be a passphrase)